web/src/main/java/com/futurewei/alcor/web/entity/route/Router.java

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.web.entity.route;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.common.entity.CustomerResource;
+import lombok.Data;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import java.util.List;
+
+@Data
+public class Router extends CustomerResource {
+
+    @JsonProperty("routetable")
+    private List<RouteTable> routeTables;
+
+    // store vpc_id
+    @JsonProperty("owner")
+    private String owner;
+
+    // store vpc_id
+    @JsonProperty("router_extra_attribute_id")
+    private String routerExtraAttributeId;
+
+    // store subnet_gateway_port_id
+    @JsonProperty("ports")
+    private List<String> ports;
+
+    @JsonProperty("tenant_id")
+    private String tenantId;
+
+    @JsonProperty("admin_state_up")
+    private boolean adminStateUp;
+
+    @JsonProperty("status")
+    private String status;
+
+    @CreatedDate
```

Comment:
created_at and updated_at are included in CustomerResource, right?

Reply:
Not included in CustomerResource, so we have to add them

Reply:
Got it.